src/biotite/sequence/io/fasta/convert.py

```diff
@@ -16,7 +16,7 @@
            "get_alignment", "set_alignment"]
 
 
-def get_sequence(fasta_file, header=None):
+def get_sequence(fasta_file, seq_type=None, header=None):
```

Comment:
I think for backwards compatibility reasons the new parameter should be the last parameter in the list, e.g. in case someone uses the `header` as positional argument in their code. Although Biotite is still in `0.x`version, where compatibility is not strictly required, I would prefer the *more compatible way*, if there is no clear advantage of the other option.

Reply:
I placed the arguments based on gut feeling. However, I agree that compatibility should take priority here.